frontend/javascripts/oxalis/view/right-menu/meshes_view.js

```diff
@@ -163,6 +213,49 @@ class MeshesView extends React.Component<Props, State> {
       Store.dispatch(setPositionAction(seedPosition, null, false));
     };
 
+    const startComputingMeshfile = async () => {
+      const availableResolutions = getResolutions(this.props.dataset);
+      const meshfileResolution =
+        availableResolutions.find(res => _.isEqual(res, defaultMeshfileGenerationResolution)) !=
+        null
+          ? defaultMeshfileGenerationResolution
+          : this.props.activeResolution;
```

Comment:
There are two small problems with this code:
- if the mag 4-4-2 exists, it is not used, even though it would probably be a good idea to use it
- if the `defaultMeshfileGenerationResolution` mag does not exist, this simply defaults to the active resolution (which might be mag-1 which is resource-intense)

Both issues can be resolved with `getDatasetResolutionInfo(this.props.dataset).getIndexOrClosestHigherIndex(defaultMeshfileGenerationResolutionIndex)` with `const defaultMeshfileGenerationResolutionIndex = 2` (because 2**2 = mag 4).